hack/gen_topology/gen_topo.py

```diff
@@ -25,8 +25,8 @@
 
 ----- output example begins ------
 #1. define containers, container format:
-# container, <containerId>, <limitCPU>, <usedCPU>, <reqCPU>, <limityMem>, <usedMem>, <reqMem>, <limitQPS>, <usedQPS>;
-container, containerC, 300, 180, 100, 400, 350, 250, 100, 80
+# container, <containerId>, <limitCPU>, <usedCPU>, <reqCPU>, <limityMem>, <usedMem>, <reqMem>, <limitQPS>, <usedQPS>, <limitResponseTime>, <usedResponseTime>;
```

Comment:
shall we change `ResponseTime` to `Latency`?

Reply:
Response time is used everywhere else.